ray-operator/controllers/ray/common/route.go

```diff
@@ -0,0 +1,84 @@
+package common
+
+import (
+	"github.com/ray-project/kuberay/ray-operator/controllers/ray/utils"
+
+	routev1 "github.com/openshift/api/route/v1"
+	rayv1alpha1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// BuildRouteForHeadService Builds the Route (OpenShift) for head service dashboard.
+// This is used to expose dashboard and remote submit service apis or external traffic.
+func BuildRouteForHeadService(cluster rayv1alpha1.RayCluster) (*routev1.Route, error) {
+	labels := map[string]string{
+		RayClusterLabelKey:                cluster.Name,
+		RayIDLabelKey:                     utils.GenerateIdentifier(cluster.Name, rayv1alpha1.HeadNode),
+		KubernetesApplicationNameLabelKey: ApplicationName,
+		KubernetesCreatedByLabelKey:       ComponentName,
+	}
+
+	// Copy other ingress configurations from cluster annotations to provide a generic way
```

Comment:
Why do we need this as I mentioned before https://github.com/ray-project/kuberay/pull/1185/files#r1255235786? Does OpenShift route use `kubernetes.io/ingress.class` annotation to specify the ingress class?

cc @anishasthana

Reply:
Removed